### PR TITLE
DP-7591 Port SVG Search icon to React

### PR DESCRIPTION
**Pull request by iansholtys**
**Description**
Adds the SVG Search icon. No readme was provided for the original component, as it's self-explanatory.

**JIRA**
https://jira.state.ma.us/browse/DP-7591

**To Test**
- [x] Pull the branch down and run `npm start`
- [x] Navigate to http://localhost:6006/ and confirm that the Atoms > Icons > SVG Search component exists and matches the appearance of [Mayflower](https://mayflower.digital.mass.gov/?p=atoms-svg-search)

**Screenshot**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export { SocialIcon } from './components/SocialIcon';
 export { Button } from './components/atoms/buttons/Button';
 export { SiteLogo } from './components/atoms/media/site-logo/SiteLogo';
 export { LatLonGlobe } from './components/atoms/icons/LatLonGlobe/LatLonGlobe';
+export { SvgSearch } from './components/atoms/icons/SvgSearch';
```
Comment (avrilpearl) [inline src/index.js]: @iansholtys  - if you update the file name per comment above this is fine.

File: src/components/atoms/icons/SvgSearch/SvgSearch.stories.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import SvgSearch from './index';
```
Comment (avrilpearl) [inline src/components/atoms/icons/SvgSearch/SvgSearch.stories.js]: @iansholtys per comment below - point to updated file name

File: src/components/atoms/icons/SvgSearch/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (mrjmd) [reply src/components/atoms/icons/SvgSearch/index.js]: This new naming was my suggestion l, will shorten our imports

File: src/index.js (hunk 1 above)
Comment (mrjmd) [reply src/index.js]: it will find index.js automatically

**Pull request merged**